### ISSUE TITLE
tomshardware.com (anti adblock lockout message)

### ISSUE THREAD

**Issue by an94baller** [uBlockOrigin/uAssets#8030]
### URL(s) where the issue occurs

https://www.tomshardware.com/

https://www.tomshardware.com/news/asus-is-making-gundam-styled-pc-parts-for-the-japanese-market

### Describe the issue

There's a javascript code that blocks page viewing and locks user to a specific URL in order to turn off adblocker

Lockout page: https://www.tomshardware.com/cc.html

### Screenshot(s)

https://i.imgur.com/y7ohZFL.png

https://www.tomshardware.com/cc.html (lockout page)

### Versions

- Browser/version: Firefox on Windows 10 / version 81.0.2
- uBlock Origin version: uBlock Origin v1.30.2

### Settings

Default blocking list

### Notes

Disabling scripts entirely by means of NoScript does "fix" the issue but renders the site unusable

**Comment by uBlock-user** [comment]
Test on default settings only. Blocking CDN scripts causes the anit-adblock to appear.

**Comment by an94baller** [comment]
> 
> 
> Test on default settings only. Blocking CDN scripts causes the anti-adblock to appear.

I experienced this with noscript disabled

I mentioned noscript as a temporary fix that I discovered

**Comment by uBlock-user** [comment]
Something is blocking CDN scripts, could be another extensions or something external which is something for you investigate. A fix is only added if reproduced on default settings and without any external interference as mentioned above.

**Comment by grande-mephistopheles** [comment]
@uBlock-user I'm having the same issue.  I'm also using the [Decentraleyes](https://decentraleyes.org/) addon for Firefox 82.0
Is that the reason?

**Comment by uBlock-user** [comment]
Test on default settings on uBO with all other extensions (except uBO & DecentralEyes) disabled.

**Comment by grande-mephistopheles** [comment]
@uBlock-user I've tested on a new profile. Does not trigger the anti-adblock when on normal mode. When 3rd-party scripts and frames are disabled (aka medium mode) then it triggers.

**Comment by grande-mephistopheles** [comment]
@uBlock-user actually, just disabling 3rd-party script is enough to trigger it

**Comment by gwarser** [comment]
"noop" `futurecdn.net`.

**Comment by uBlock-user** [comment]
Yes, that's what I mentioned before in https://github.com/uBlockOrigin/uAssets/issues/8030#issuecomment-707897072

**Comment by grande-mephistopheles** [comment]
> Yes, that's what I mentioned before in [#8030 (comment)](https://github.com/uBlockOrigin/uAssets/issues/8030#issuecomment-707897072)
So a global "noop" futurecdn.net or only for tomshardware.com ?

**Comment by grande-mephistopheles** [comment]
@gwarser @uBlock-user okay I got it solved. you actulay need to "noop" three sites
add these three lines to your rules list:
```
tomshardware.com futurecdn.net * noop
tomshardware.com footprint.net * noop
tomshardware.com future.net.uk * noop
```

**Comment by firepainting** [comment]
@grande-mephistopheles Thanks that worked for me.

What generally is the fastest/easiest way to identify the lines I should put in "My Rules"? I'm a noob on that interface. So far I've been using the graphical columns only (assigning gray and red colors as an "advanced user").

**Comment by okiehsch** [comment]
> So far I've been using the graphical columns only

That's the easiest way, you should know what you are doing if you use dynamic filtering if you don't you can easily bork your whole setup.
That is the reason why you have to manually enable it in the settings and why it links to a **required** reading page.
https://github.com/gorhill/uBlock/wiki/Advanced-user-features
